packages/client/src/monaco-language-client.ts

```diff
@@ -52,28 +54,28 @@ export class MonacoLanguageClient extends BaseLanguageClient {
 
         // Hack because vscode-language client rejects the whole registration block if one capability registration has no associated client feature registered
         // Some language servers still send the registration even though the client says it doesn't support it
-        const originalHandleRegistrationRequest: (params: RegistrationParams) => Promise<void> = this['handleRegistrationRequest'].bind(this)
+        const originalHandleRegistrationRequest: (params: RegistrationParams) => Promise<void> = this['handleRegistrationRequest'].bind(this);
         this['handleRegistrationRequest'] = (params: RegistrationParams) => {
             originalHandleRegistrationRequest({
                 ...params,
                 registrations: params.registrations.filter(registration => this.getFeature(<any>registration.method) != null)
-            })
-        }
-        const originalHandleUnregistrationRequest: (params: UnregistrationParams) => Promise<void> = this['handleUnregistrationRequest'].bind(this)
-        this['handleUnregistrationRequest'] = (params: UnregistrationParams) => {
+            });
+        };
+        const originalHandleUnregistrationRequest: (params: UnregistrationParams) => Promise<void> = this['handleRegistrationRequest'].bind(this);
+        this['handleRegistrationRequest'] = (params: UnregistrationParams) => {
```

Comment:
I don't think it should be changed

Reply:
Sorry, stupid copy paste issue, when I changed the code back (manually)